test/models/content_view_version_test.rb

```diff
@@ -36,10 +36,14 @@ def test_promotable_without_environments2
 
     def test_of_version
       version = @cvv
+      content_view = version.content_view
+      content_view.versions << @cvv_minor
+
       assert_equal [version], version.content_view.versions.for_version("1.0")
       assert_equal [version], version.content_view.versions.for_version("1")
       assert_equal [version], version.content_view.versions.for_version(1)
       assert_equal [version], version.content_view.versions.for_version(1.0)
+      assert_equal [minor_version], version.content_view.versions.for_version(1.1)
```

Comment:
@neoseele hey sorry, I edited my comment after I posted it, you may have seen the original one. This should be
```assert_equal [@cvv_minor], version.content_view.versions.for_version(1.1)```